online-shop/src/app/product-list/product-list.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="shop-title">
+  <h2>Products</h2>
+</div>
+
+<div class="table-products">
```

Comment:
try using BEM for CSS 
here is how to use it with scss: https://www.pullrequest.com/blog/bem-and-sass-whats-the-difference-and-how-to-use-them/